app/config/security.yml

```diff
@@ -172,6 +172,12 @@ security:
         ROLE_ADMIN_BOARD_MEMBER_ROLES:
             - ROLE_APP_ADMIN_ROLE_ALL
 
+        ROLE_ADMIN_REPORTS:
+            - ROLE_APP_ADMIN_REPORT_APPROVE
+            - ROLE_APP_ADMIN_REPORT_VIEW
+            - ROLE_APP_ADMIN_REPORT_LIST
+            - ROLE_APP_ADMIN_REPORT_ALL
```

Comment:
`ROLE_APP_ADMIN_REPORT_ALL` implicetely includes  `LIST` and `VIEW` roles.

Reply:
Thank you I'll remove them ;-)